src/proto/thread_telemetry.proto

```diff
@@ -115,6 +115,8 @@ message TelemetryData {
     optional uint32 neighbor_table_size = 15;
     optional int32 instant_rssi = 16;
     optional uint64 extended_pan_id = 17;
+    // Indicates the number peer BR in Thread mesh network (from network data)
+    optional uint32 peer_br_count = 18;
```

Comment:
nit: Probably we should use different names for `TelemetryData.peer_br_count` and `InfraLinkInfo.peer_br_count` so they are more distinguishable without looking at the documentation.

Reply:
I overlooked so I thought `peer_br_count` is in message `TelemetryData`. Since it's part of message `WpanTopoFull` I think it should be clear enough.